src/storage/migrations/11-fightsystem.ts

```diff
@@ -0,0 +1,29 @@
+import { sql, type Kysely } from "kysely";
+import { createUpdatedAtTrigger } from "@/storage/migrations/10-loot-attributes.js";
```

Comment:
Die funktion kannste ruhig rüberkopieren

Reply:
oder in eine sql- migration- helper auslagern?

Reply:
Migrationen sollten jeweils komplett self-contained und "in der Zeit eingefroren" sein. Sonst wird irgendwann mal irgendwas minimales in dem Shared-Teil geändert und dann schlagen alte migrationen fehl, wenn man ein neues Dev-System aufsetzt